tests/Unit/Css/CssDocumentTest.php

```diff
@@ -22,6 +24,11 @@ final class CssDocumentTest extends TestCase
         . '  font-family: "Foo Sans";' . "\n"
         . '  src: url("/foo-sans.woff2") format("woff2");' . "\n}";
 
+    private static function createDebugSubject(string $css): CssDocument
```

Comment:
Please let's not make this static (as there is no need to make it static).

Reply:
Changed.